### PR TITLE
Update pod-lifecycle.md

### PR DESCRIPTION
**Pull request by lopes22**
Correct grammar.

### CODE REVIEW

**Comment by lopes22** [comment]
signed CLA

File: content/en/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ about when the container entered the `Running` state.
 
 ### `Terminated` {#container-state-terminated}
 
-A container in the `Terminated` state has begin execution and has then either run to
+A container in the `Terminated` state had began execution and then has either run to
 completion or has failed for some reason. When you use `kubectl` to query a Pod with
 a container that is `Terminated`, you see a reason, and exit code, and the start and
 finish time for that container's period of execution.
```
Comment (zacharysarah) [inline content/en/docs/concepts/workloads/pods/pod-lifecycle.md:130]:
```suggestion
A container in the `Terminated` state began execution and then either ran to
completion or failed for some reason. When you use `kubectl` to query a Pod with
a container that is `Terminated`, you see a reason, an exit code, and the start and
finish time for that container's period of execution.
```

**Pull request merged**